src/client/scripts/views/CreateAccount/NeedHousingView.js

```diff
@@ -22,29 +21,123 @@ export class NeedHousingView {
         content.innerHTML = `
             <h2>Create Account - Need Housing</h2>
             <form id="needHousingForm">
-                <label for="firstName">First Name:</label>
-                <input type="text" id="firstName" name="firstName" required><br><br>
-                
-                <label for="lastName">Last Name:</label>
-                <input type="text" id="lastName" name="lastName" required><br><br>
-                
-                <label for="email">Email:</label>
-                <input type="email" id="email" name="email" required><br><br>
-                
-                <label for="phone">Phone:</label>
-                <input type="tel" id="phone" name="phone" required><br><br>
-                
                 <label for="housingPreferences">Housing Preferences:</label>
-                <textarea id="housingPreferences" name="housingPreferences" required></textarea><br><br>
-                
+                <div class="input-box">
+                    <textarea id="housingPreferences" name="housingPreferences" rows="9" style="width: 360px; height: 354.45px;" required></textarea>
+                    <div class="ovals">
+                        <div class="row">
+                            <button class="oval-button">Cities (comma-separated)</button>
+                        </div>
+                        <div class="row">
+                            <button class="oval-button">Min-rent</button>
+                            <span>-</span>
+                            <button class="oval-button">Max-rent</button>
+                        </div>
+                        <div class="row">
+                            <button class="oval-button">Min-occupants</button>
+                            <span>-</span>
+                            <button class="oval-button">Max-occupants</button>
+                        </div>
+                        <div class="row">
```

Comment:
We don't have a multi-select dropdown as shown in the Figma, so I made a workaround. Use the CheckboxGroup component instead:
![image](https://github.com/ashleybhandari/team-friendship/assets/99090595/f7d6b8f8-9b76-4c34-b677-a6960dd19244)

Reply:
This should be used for everything on this page except for Cities, Min/Max rent, and Min/Max occupants

Reply:
I see, gotcha thank you I will